defect-tracker-data-common/src/main/java/com/defect/tracker/data/entities/Employee.java

```diff
@@ -33,7 +33,28 @@
 	private String contactNumber;
 	private String nic;
 	private String image;
+	private String verification;
```

Comment:
write the validation

Reply:
no need to validate on entity it is validated on DTO